packages/data-context/src/sources/migration/utils.ts

```diff
@@ -0,0 +1,9 @@
+import _ from 'lodash'
+
+export const isDefaultSupportFile = (supportFile: string) => {
+  if (_.isNil(supportFile) || !_.isBoolean(supportFile) && supportFile.match(/^cypress\/support($|\/index($|\.(ts|js|coffee)$))/)) {
```

Comment:
I know this code was lifted, but what happens if the user has `./cypress/support/index.js` (prefixed with "./"). This is the default file but the regex wouldn't match

Reply:
Maybe we should update the regex for something like `(^|.+/)cypress\/support($|\/index($|\.(ts|js|coffee)$))` what do you think?

Reply:
[2e5b5d6](https://github.com/cypress-io/cypress/pull/22015/commits/2e5b5d6c9bda0efd3245056f9746e920e00a5b93)